packages/ember-glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -0,0 +1,545 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { strip } from '../../utils/abstract-test-case';
+import { set } from 'ember-metal/property_set';
+import { get } from 'ember-metal/property_get';
+import setProperties from 'ember-metal/set_properties';
+import Component from 'ember-views/components/component';
+import EmberObject from 'ember-runtime/system/object';
+import { A as emberA } from 'ember-runtime/system/native_array';
+
+moduleFor('Helpers test: {{unbound}}', class extends RenderingTest {
+
+  ['@htmlbars it should not update the value if the parent view doesn\'t rerender']() {
+    this.render(`{{unbound foo}} {{unbound bar}}`, {
+      foo: 'BORK',
+      barBinding: 'foo'
+    });
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'OOF'));
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'BORK'));
+
+    this.assertText('BORK BORK');
+  }
+
+  ['@htmlbars should be able to output a property without binding']() {
+    this.render(`<div id="first">{{unbound content.anUnboundString}}</div>`, {
+      content: EmberObject.create({
+        anUnboundString: 'No spans here, son.'
+      })
+    });
+
+    this.assertText('No spans here, son.');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('No spans here, son.');
+
+    this.runTask(() => set(this.context, 'content.anUnboundString', 'HEY'));
+
+    this.assertText('No spans here, son.');
+
+    this.runTask(() => set(this.context, 'content', EmberObject.create({
+      anUnboundString: 'No spans here, son.'
+    })));
+
+    this.assertText('No spans here, son.');
+  }
+
+  ['@htmlbars should be able to use unbound helper in #each helper']() {
```

Comment:
Port of https://github.com/emberjs/ember.js/blob/bf067f7370c3c3381352d68ef5dfe55d1178c6f7/packages/ember-htmlbars/tests/helpers/unbound_test.js#L543-L553